docs/usage/exporting_models.md

```diff
@@ -43,6 +46,34 @@ print(m.model_dump(include={'foo', 'bar'}))
 #> {'foo': 'hello', 'bar': {'whatever': 123}}
 print(m.model_dump(exclude={'foo', 'bar'}))
 #> {'banana': 3.14}
+print(m.model_dump(by_alias=True))
+#> {'banana': 3.14, 'foo_alias': 'hello', 'bar': {'whatever': 123}}
+print(FooBarModel(foo='hello', bar={'whatever': 123}).model_dump(exclude_unset=True))
+#> {'foo': 'hello', 'bar': {'whatever': 123}}
+print(
+    FooBarModel(banana=1.1, foo='hello', bar={'whatever': 123}).model_dump(
+        exclude_defaults=True
+    )
+)
+#> {'foo': 'hello', 'bar': {'whatever': 123}}
+print(FooBarModel(foo='hello', bar={'whatever': 123}).model_dump(exclude_defaults=True))
+#> {'foo': 'hello', 'bar': {'whatever': 123}}
+print(
+    FooBarModel(banana=None, foo='hello', bar={'whatever': 123}).model_dump(
+        exclude_none=True
+    )
+)
+#> {'foo': 'hello', 'bar': {'whatever': 123}}
+
+
+class Model(BaseModel):
+    x: List[Json[Any]]
+
+
+print(Model(x=['{"a": 1}', '[1, 2]']).model_dump())
+#> {'x': [{'a': 1}, [1, 2]]}
+print(Model(x=['{"a": 1}', '[1, 2]']).model_dump(round_trip=True))
+#> {'x': ['{"a":1}', '[1,2]']}
```

Comment:
Isn't this a bit too much code for this section? - If you disagree, just resolve this thread please.